upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -681,7 +681,7 @@ if [[ $state_recorded == "0" ]]; then
     export PDSH_SSH_ARGS_APPEND="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
     #shellcheck disable=SC2046
     rpm --force -Uvh $(find $CSM_ARTI_DIR/rpm/cray/csm/ -name \*csm-testing\*.rpm | sort -V | tail -1)
-    /opt/cray/tests/install/ncn/scripts/validate-bootraid-artifacts.sh
+    /opt/cray/tests/install/ncn/scripts/check_bootloader.sh
```

Comment:
Not sure if this will run on every node.. but I also think we don't need this to run at all. The upgrade isn't going to use the disk bootloader anyway.

The previous script did check BSS, but that's no longer necessary and this script doesn't do that check.